.github/workflows/python.yaml

```diff
@@ -43,6 +43,8 @@ jobs:
           aws-region: us-east-1
       - name: Copy build output to public S3
         run: |
+          aws s3 cp python/cpython/python.wasm                 s3://timecraft/python/nightly/python.wasm
```

Comment:
Should this be gated to only run on `main`? Otherwise those files will also be updated by commits on branches.

Also nitpick, maybe `/python/main/python.*` instead of nightly? It's not like we do a daily build.